src/lib/comms/sol-network-impl-linux.c

```diff
@@ -580,13 +580,13 @@ sol_network_link_set_status(uint16_t link_index, unsigned int changes, unsigned
     return true;
 }
 
-SOL_API bool
+SOL_API int
 sol_network_link_up(uint16_t link_index)
 {
     return sol_network_link_set_status(link_index, IFF_UP, IFF_UP);
```

Comment:
In this case, should I change the sol_network_link_status( ) or should I keep it as the same way and change sol_network_link_up( ) and change sol_network_link_down( )?